packages/cdktf/lib/terraform-resource.ts

```diff
@@ -90,6 +96,7 @@ export class TerraformResource
   public provisioners?: Array<
     FileProvisioner | LocalExecProvisioner | RemoteExecProvisioner
   >;
+  public imported?: TerraformResourceImport;
```

Comment:
We should make this private and prefix an underscore (which JSII will required). Else we'd also have a collision on this attribute name if resources have an `imported` attribute. Or are there reasons for this to be public?